MusicNotationKit/MusicNotationKit/Note.swift

```diff
@@ -53,3 +53,34 @@ extension Note: NoteCollection {
 	
 	internal var noteCount: Int { return 1 }
 }
+
+extension Note: CustomDebugStringConvertible {
+	public var debugDescription: String {
+		
+		// Tones are handled in two different ways. Single notes
+		// do not contain collection notation, whereas chords do.
+		var tones:String = ""
+		if (self.tones.count == 1) {
+			tones = String(self.tones[0])
+		} else if (self.tones.count > 1) {
+			tones = String(self.tones)
+		}
+		
```

Comment:
- No parentheses in `if/else if`
- No `self.` needed
- Use string interpolation